README.md

```diff
@@ -64,6 +60,8 @@ The templates the module will use are:
   with the wrong token. It doesn't get passed any variable.
 
 You can find and change the default templates [here](https://github.com/matrix-org/synapse-email-account-validity/tree/main/email_account_validity/templates).
+Admins can install custom templates either by changing the default ones directly, or by
+configuring Synapse with a custom template directory that contains the custom templates.
```

Comment:
I don't think we should fix it as part of this PR, but we need to sort out the template handling.
 * How does an admin change the default templates, given that, according to this they live on github in the matrix-org account?
 * How exactly does an admin configure Synapse with a custom template directory?
 * Even if you do configure a custom template directory, are you sure that these templates will get read from it?
 * Indeed, my understanding is that to get this module working at all, you have to copy the templates into an undocumented directory in the depths of the synapse codebase?

Reply:
Answering at least one of these questions is probably out of the scope of this (and more in the scope of how Synapse handles Synapse more broadly.

> Indeed, my understanding is that to get this module working at all, you have to copy the templates into an undocumented directory in the depths of the synapse codebase?

Right, that's not the case, I'll add a line to the documentation to make this clear.

Reply:
> Answering at least one of these questions is probably out of the scope of this (and more in the scope of how Synapse handles Synapse more broadly

maybe, but as long as we're adding more documentation, it could do with being right.